packages/visualizations/src/components/MapPoi/utils.ts

```diff
@@ -26,63 +27,100 @@ export const getMapSources = (sources: PoiMapData['sources']): StyleSpecificatio
     return sources;
 };
 
-// Only circle layers are supported
-export const getMapLayers = (layers?: Layer[]): CircleLayerSpecification[] => {
-    if (!layers) return [];
+const getBaseMapLayerConfiguration = (layer: Layer) => {
+    const { id, source, sourceLayer } = layer;
+    return {
+        id,
+        source,
+        ...(sourceLayer ? { 'source-layer': sourceLayer } : undefined),
+        filter: ['==', ['geometry-type'], 'Point'],
```

Comment:
Seems to work well with `null`
```suggestion
        ...(sourceLayer ? { 'source-layer': sourceLayer } : null),
```